src/main/webapp/less/includes/theme.less

```diff
@@ -16,79 +16,1048 @@
+            .flc-pager-pageLink {
+                a {
+                    color: @color1 !important;
+                    text-decoration: none !important;
+                }
+                a.fl-pager-currentPage {
+                    color: @white !important;
+                }
+            }
+        }
+    }
+    .container-list {
+        -moz-transform: translate3d(15px,0,0);
+        -ms-transform: translate3d(15px,0,0);
+        -o-transform: translate3d(15px,0,0);
+        -webkit-transform: translate3d(15px,0,0);
+        background: white;
+        min-height: 100%;
+        position: absolute;
+        top: 50px;
+        transform: translate3d(15px,0,0);
+        width:@width100;
+        z-index: 1;
+        .pagination>li>a {
+            color: @color1 !important;
+        }
+        .pagination>li>span {
+            color: @color1 !important;
+        }
+        .pagination>.active>a {
+            background-color: @color1;
+            border-color: @color1;
+            color: @white !important;
+            cursor: default;
+            z-index: 2;
+            &:hover {
+                background-color: @color1;
+                border-color: @color1;
+                color: @white !important;
+                cursor: default;
+                z-index: 2;
+            }
+            &:focus {
+                background-color: @color1;
+                border-color: @color1;
+                color: @white !important;
+                cursor: default;
+                z-index: 2;
+            }
+        }
+        .pagination>.active>span {
+            background-color: @color1;
+            border-color: @color1;
+            color: @white !important;
+            cursor: default;
+            z-index: 2;
+            &:hover {
+                background-color: @color1;
+                border-color: @color1;
+                color: @white !important;
+                cursor: default;
+                z-index: 2;
+            }
+            &:focus {
+                background-color: @color1;
+                border-color: @color1;
+                color: @white !important;
+                cursor: default;
+                z-index: 2;
+            }
+        }
+        .pagination>.flc-pager-pageLink>.fl-pager-currentPage {
+            background-color: @color1 !important;
+            border-color: @color1 !important;
+            color: @white !important;
+            cursor: default;
+            z-index: 2;
+            &:hover {
+                background-color: @color1 !important;
+                border-color: @color1 !important;
+                color: @white !important;
+                cursor: default;
+                z-index: 2;
+            }
+            &:focus {
+                background-color: @color1 !important;
+                border-color: @color1 !important;
+                color: @white !important;
+                cursor: default;
+                z-index: 2;
+            }
```

Comment:
@ChristianMurphy 
300-464 : converts Selected Message's "fixed content" to "fluid content"
I know that `!important` is a question, but when you've got newsletters designers who are using `nested tables with html inlines styles***` such : 

`<TABLE WIDTH="1024" HEIGHT="960"><TR><TD><TABLE WIDTH="500" HEIGHT="30">...bunch of nested tables`

`***` it's the only ways of doing things in html email rendering...

You end up with something like this :

![alt text][img]

[img]: https://issues.jasig.org/secure/attachment/15416/emailpreview-displaying-none-fluid-message-eg-a%20newsletter-beforetheCSSimportantpatch.png "The fixed newsletter overflow crappy display"

but with !important statement (it's scoped only on the message div), I've got this:


![alt text][img2]

[img2]: https://issues.jasig.org/secure/attachment/15246/fixedcontentConverttoFluid%20content.png "The fixed newsletter convert to fluid display"

without injecting any parsing JS that need to parse the message content which will be **a security and integrity issue** :warning: :collision: :no_entry_sign: ...
for the moment, I know that Google's statement for removing `!important` of any CSS code is an advice and because there are someothers none [Blink ](https://en.wikipedia.org/wiki/Blink_(web_engine))rendering engine like Chrome that do not have the same selectors priority hierarchy, like [Webkit ](https://en.wikipedia.org/wiki/WebKit)Safari I do not mention [Trident](https://en.wikipedia.org/wiki/Trident_(layout_engine)), `!important` is still very usefull in some cases... :smiley:
 Trust me... :wink: